libraries/RC_Channel/RC_Channels_VarInfo.h

```diff
@@ -92,5 +92,12 @@ const AP_Param::GroupInfo RC_Channels::var_info[] = {
     // @Bitmask: 0:Ignore RC Receiver, 1:Ignore MAVLink Overrides, 2:Ignore Receiver Failsafe, 3:FPort Pad, 4:Log RC input bytes, 5:Arming check throttle for 0 input, 6:Skip the arming check for neutral Roll/Pitch/Yay sticks, 7:Allow Switch reverse
     AP_GROUPINFO("_OPTIONS", 33, RC_CHANNELS_SUBCLASS, _options, (uint32_t)RC_Channels::Option::ARMING_CHECK_THROTTLE),
 
+    // @Param: _PROTOCOLS
+    // @DisplayName: Enabled RC protocols
+    // @Description: This is the set of enabled RC protocols. You can use this to enable detection of only specific types of RC receivers. This can be used to avoid issues with incorrect protocol detection. Set to 1 to allow all protocols.
+    // @User: Advanced
+    // @Bitmask: 0:All,1:PPM,2:IBUS,3:SBUS,4:SBUS_NI,5:DSM,6:SUMD,7:SRXL,8:SRXL2,9:CRSF,10:ST24,11:FPORT
+    AP_GROUPINFO("_PROTOCOLS", 34, RC_CHANNELS_SUBCLASS, _protocols, 1),
```

Comment:
I think this is a great improvement. I have to say from a user perspective it would make more sense to me to simply require the user to specify the protocol in SERIALn_PROTOCOL.

Reply:
I want it for pulse decoding and IOMCU decoding too